README.md

```diff
@@ -2,31 +2,96 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Describe the components of a web application framework.
+- Describe the history of the Flask web microframework.
+
+---
+
+## Key Vocab
+
+- **Web Framework**: software that is designed to support the development of web
+  applications. Web frameworks provide built-in tools for generating web
+  servers, turning Python objects into HTML, and more.
+- **Extension**: a package or module that adds functionality to a Flask
+  application that it does not have by default.
+- **Web Server**: a combination of software and hardware that uses Hypertext
+  Transfer Protocol (HTTP) and other protocols to respond to requests made over
+  the internet.
+
+---
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+In this module, we will learn how to build Python-based web applications with
+[Flask](https://flask.palletsprojects.com/en/3.0.x/), a web microframework.
+
+A web framework is a tool that is designed to support the development of web
+applications. This is similar to React, though true web frameworks provide a
+degree of scaffolding to direct you in the construction of your project. Certain
+frameworks like [Django](https://www.djangoproject.com/) do everything under the
+sun: they give you the power of Python, an ORM, a web server, authentication,
```

Comment:
while most of this is evident/understandable what you're talking about, I don't know if students will know what an ORM is yet at this point so this may be confusing

Reply:
add brief explanation of ORM